src/Framework/Runner/DataTypes.cs

```diff
@@ -1,32 +1,48 @@
 ﻿using System;
-using System.CodeDom;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Collections.Specialized;
 using System.ComponentModel;
 using System.IO;
 using System.Linq;
 using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
-using System.Text;
-using System.Threading;
 using System.Xml.Serialization;
-using Microsoft.Practices.Prism.ViewModel;
 
 namespace RTF.Framework
 {
     [Serializable]
     public class AssemblyData : IAssemblyData
     {
         private bool? _shouldRun = true;
-        private ObservableCollection<ITestGroup> _sortingGroup;
+        private bool _isNodeExpanded;
```

Comment:
style thing - I think we prefer to not use `_ ` notation for fields - generally we just name the field camelCase and the property PascalCase.

Reply:
I agree with that, was just trying to follow "when in Rome" rule for consistency. But I am happy to fixup the whole file if you prefer. Let me know